crates/matrix-sdk-crypto/src/store/integration_tests.rs

```diff
@@ -1218,8 +1218,9 @@ macro_rules! cryptostore_integration_tests {
             }
 
             #[async_test]
+            #[ignore]
```

Comment:
this will not run the test, so admittedly yeah, it won't make it flaky anymore :sweat_smile:

Reply:
Sorry, fixed in 9dab41e118